src/components/TabGroup/__tests__/__snapshots__/TabGroup.test.tsx.snap

```diff
@@ -36,11 +36,11 @@ exports[`TabGroup renders correctly 1`] = `
     class="cache-zkzsno e7ah3bn0"
   >
     <div
-      class=" cache-1inpyg4 ex5i7pr1"
+      class=" cache-1inpyg4 e1iqq30b1"
       role="tablist"
     />
     <div
-      class="cache-1rqct99 ex5i7pr0"
+      class="cache-1rqct99 e1iqq30b0"
```

Comment:
on a une idee de ce que c'est ce bruit?

Reply:
Do you know why maybe @adriengibrat  ?

Reply:
Because one test was drop by @chambo-e :)

Reply:
Otherwise it's javascript -> typescript that impact this (during building phase)

Reply:
No idea, I've read & assumed className hashes were only dependent of css content....
Don't look like it's the case here.